core/source/Time.mint

```diff
@@ -21,185 +36,617 @@ module Time {
     `
   }
 
+  /* CONVERTING ------------------------------------------------------------- */
+
+  /*
+  Returns the UNIX Timestamp (in milliseconds) of the given time.
+
+    Time.toUnix(Time.utcDate(2006, 1, 2)) == 1136160000000
+  */
+  fun toUnix (time : Time) : Number {
+    `#{time}.getTime()`
+  }
+
+  /* CREATING ---------------------------------------------------------------- */
+
+  /*
+  Returns the time respective to the given UNIX Timestamp (in milliseconds).
+
+    Time.unix(1136160000000) == Time.utcDate(2006, 1, 2)
+  */
+  fun unix (timestamp : Number) : Time {
+    `new Date(#{timestamp})`
+  }
+
+  /*
+  Returns a new time from the given parameters.
+
+    Time.utc(2018, 4, 5)
+  */
+  fun utc (
+    year : Number,
+    month : Number,
+    day : Number,
+    hour : Number,
+    minute : Number,
+    second : Number,
+    millisecond : Number
```

Comment:
All of these should be optional.

Reply:
I agree, but Mint doesn't have optional arguments (yet).

Reply:
Doc example should be updated then.

Reply:
Where do you see that in the docs? There is `Time.utc` with all parameters and `Time.utcDate` with just year, month, day.

Reply:
`Time.utc(2018, 4, 5)`